code/game/machinery/doors/airlock.dm

```diff
@@ -571,8 +571,12 @@
 			frame_state = AIRLOCK_FRAME_OPEN
 			// If we're open we layer the bit below us "above" any mobs so they can walk through
 			if(!short_rendering)
-				. += mutable_appearance(icon, "open_bottom", ABOVE_MOB_LAYER, appearance_flags = KEEP_APART)
-				. += emissive_blocker(icon, "open_bottom", src, ABOVE_MOB_LAYER)
+				var/mutable_appearance/bottom = mutable_appearance(icon, "open_bottom", ABOVE_MOB_LAYER, appearance_flags = KEEP_APART)
+				bottom.alpha = 185
+				. += bottom
+				bottom = emissive_blocker(icon, "open_bottom", src, ABOVE_MOB_LAYER)
```

Comment:
these procs have alpha args, you should use them. Especially for emissive blockers if memory serves what you're doing here is gonna fuck with the applied color matrix or someodd.